src_assets/common/assets/web/troubleshooting.html

```diff
@@ -128,6 +140,8 @@ <h2 id="logs">{{ $t('troubleshooting.logs') }}</h2>
           unpairAllPressed: false,
           unpairAllStatus: null,
           restartPressed: false,
+          showApplyMessage: false,
+          clients: [],
           logs: 'Loading...',
           logFilter: null,
           logInterval: null,
```

Comment:
Can you alphabetize these? clients through logInterval

Reply:
Done 👍

I misunderstood what you meant initially, so the device list is now alphabetical too... which is better than being unsorted. 😅